plugins/inventory/efficientip.py

```diff
@@ -0,0 +1,367 @@
+# -*- Mode: Python; python-indent-offset: 4 -*-
+#
+# Time-stamp: <2020-07-07 18:18:40 alex>
+#
+
+"""
+ EfficientIP inventory for ansible.
+ SOLIDserver Device Manager connection to gather devices filtered.
+"""
```

Comment:
```suggestion
# -*- Mode: Python; python-indent-offset: 4 -*-
#
# Time-stamp: <2020-07-07 18:18:40 alex>
#

"""
 EfficientIP inventory for ansible.
 SOLIDserver Device Manager connection to gather devices filtered.
"""
```
could you please make it in a more standard way? Look into other plugins, e.g. in filter/time.py
```
  1 # -*- coding: utf-8 -*-
  2 # Copyright (c) 2020, René Moser <mail@renemoser.net>
  3 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
  4
  5 from __future__ import (absolute_import, division, print_function)
  6 __metaclass__ = type
```